rosdep/python.yaml

```
@@ -8442,6 +8442,81 @@ python3-pyside2.qtwidgets:
     '*': [python3-pyside2]
     '8': null
   ubuntu: [python3-pyside2.qtwidgets]
+python3-pyside6:
+  debian:
+    bookworm:
+      pip:
+        packages: [PySide6]
+    bullseye:
+      pip:
+        packages: [PySide6]
+    sid: [libpyside6-dev]
```

```suggestion
    trixie: [libpyside6-dev]
    sid: [libpyside6-dev]
```

Add the current Debian Testing distribution as well. I would even consider dropping the `sid` definition altogether in favor of just trixie but don't mind it being there.

It's worth making the change for each of the definitions below as well.

---

Done in https://packages.debian.org/testing/python3-pyside6.qtasyncio

---

Looks like this made CI unhappy

```
    ERR: list out of alphabetical order line 8526.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8551.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8557.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8563.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8569.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8575.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8581.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8587.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8593.  'sid' should come before 'trixie'
    ERR: list out of alphabetical order line 8599.  'sid' should come before 'trixie'
```